### PR TITLE
#164771416 Submit user feedback

### PR DESCRIPTION
**Pull request by wangonya**
**What does this PR do?**
Creates a feedback endpoint to allow users post their feedback

**Description of Task to be completed?**
- [x] Add a feedback endpoint

**How should this be manually tested?**
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Using
[Postman](https://www.getpostman.com/) open create a `GET` & `POST` request to `api/v2/feedback`

**Any background context you want to provide?**
Users may want to submit comments, suggestions, questions or concerns. This implementation provides that feature.

**What are the relevant pivotal tracker stories?**
[#164771416](https://www.pivotaltracker.com/n/projects/2319972/stories/164771416)

**Screenshots (if appropriate)**
Screenshot 2019-03-21 at 14 38 37

### CODE REVIEW

**Comment by wekesa931** [comment]
Good feature and documentation.

File: app/api/v2/views/feedback.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+@path_2.route("/feedback", methods=["GET"])
+def get_all_feedback():
+    """
+        This method gets all the posted feedback
+    """
+    feedback = FeedbackModel.get_all_feedback()
+    if feedback:
+        return utils.response_fn(200, "data", feedback)
+    return utils.response_fn(200, "data", [])
+
```
Comment (willeswa) [inline app/api/v2/views/feedback.py]: Would you consider removing the empty lines?

Comment (wangonya) [reply app/api/v2/views/feedback.py]: Thanks for the suggestion. However, pep8 standards require that two blank lines be left after imports, before classes, and before decorators. @philophilo kindly you can clarify on this.

Comment (willeswa) [reply app/api/v2/views/feedback.py]: Ooh. That is news to me :).

Comment (wangonya) [reply app/api/v2/views/feedback.py]: The more you know 😄

Comment (wangonya) [reply app/api/v2/views/feedback.py]: [Here is a reference](https://pep8.org/#blank-lines) for your convenience @willeswa

**Pull request closed without merging**